src/Core/src/Config/DeprecationProxy.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Spiral\Core\Config;
+
+final class DeprecationProxy extends Proxy
```

Comment:
Add `@deprecated` annotation?

Reply:
Added `@internal`.  This question is open for further discussion.